tests/testthat/test-line_numbers.r

```diff
@@ -29,7 +29,7 @@ test_that("line numbers are found and given to reporters", {
   ### ==== EDGE CASES ====
   
   # test file with errors, e.g. unknown function
-  expect_error(.test_code('expect_toto()'), regexp = 'Error')
+  expect_warning(.test_code('expect_toto()'), regexp = 'File sourcing')
```

Comment:
A potential (pedantic) comment may be to make the quoting consistent with your change on line 44 (both single quotes or both double quotes).